docs/topics/api/addons.rst

```diff
@@ -76,6 +78,18 @@ This endpoint allows you to search through public add-ons.
     sort parameter, which is only available alone.
 
 
+.. _addon-threshold-param:
+
+    Threshold style parameters allow queries against numeric values using comparison operators.
+
+    The following is supported (examples for query parameter `foo`):
+    * greater than > (example query: ?foo=>10.1)
+    * less than < (example query: ?foo=<10.1)
+    * greater than or equal to >= (example query: ?foo=>=10.1)
+    * less than or equal to <= (example query: ?foo=<=10.1)
+    * equal to == (example query: ?foo==10.1)
```

Comment:
tbh I'm not 100% happy with the syntax.  Because queries kinda have to be of the format xxx=yyy theres always an `=` in there.  E.g. `=>` seems like it should be greater than or equal to, but then what is just greater than? (If that's actually a useful thing for comparing to a decimal value - could just only have greater|less than or equal to operators).

Reply:
could go with django's ORM style - `gt`, `lt`, `gte`, `lte`, and `=` ?

Reply:
hmm, what would an example url look like in django ORM style?

Reply:
`?foo__gt=10.1`, `?foo__lt=10.1`, `?foo__gte=10.1`, `?foo__lte=10.1`, `?foo=10.1`

Reply:
Could do.  I _did_ like the style of writing `>` directly in the url but `__gt` is more explicit.

Reply:
Could also look at how https://django-filter.readthedocs.io/en/stable/ does it

Reply:
it'd need 5 `AddonQueryParam` classes per field but that's fine.

Reply:
I agree that the double equals sign is ugly and potentially confusing. I don't love the django style either, but agree it's better.

At the risk of suggesting removing functionality, it seems like we actually only need a threshold of gte currently (i.e., must have at least x users, must have at least x rating), so maybe we don't even need all of the variations?

Reply:
its true we only need gte right now, but if it's straightforward enough (it's really little more than a lightweight layer over the ES query) I'd prefer to fully implement it - we never know when it might be useful in the future

Reply:
Fair enough. Are you going to update the patch with the django approach? If so I'll want for that to do a full review.

Reply:
When it comes to filters in json API, I've seen the following in the past (exclude OData):

```
/api/?filters[foo][eq]=100&filters[bar][gt]=50
```

and

```
/api/?foo=eq(100)&bar=gt(50)
```

Reply:
https://github.com/mozilla/addons-server/pull/17602/commits/fd2aabad7d4951dc27e75d29165ca9fdbb977e75 changes the PR to use orm __style comparison parameters.